buildshiprun/handler.go

```diff
@@ -695,3 +696,11 @@ func reportGitLabStatus(status *sdk.Status) {
 	}
 	status.CommitStatuses = make(map[string]sdk.CommitStatus)
 }
+
+func buildBranch() string {
+	branch := os.Getenv("build_branch")
```

Comment:
We can use os.Lookupenv to distinguish between empty environmental variable and unset one

Reply:
Do you need that distinction in this case? If the variable is unset or if it does not exist we need to return `master`, so if we use the LookupEnv we'll need to add another check for when the variable is empty

Reply:
No.. unset means master